lib/ringflux/plugin.rb

```diff
@@ -13,6 +13,7 @@ class Ringflux::Plugin < Adhearsion::Plugin
     database    'adhearsion', desc: 'host where the message queue is running'
     async       true        , desc: 'Asynchronously send data to InfluxDB', transform: ->(value) { value && value != "false" }
     use_ssl     nil         , desc: 'Connect to InfluxDB using SSL', transform: ->(value) { value && value != "false" }
+    retries     3           , desc: 'Number of attempts to make with the InfluxDB client'
```

Comment:
Do you need a `.to_i` transform on this?

Reply:
Didn't seem to, however are you guessing that if this were pulled from `AHN_RINGFLUX_RETRIES` environmental that it would be a string type?

Reply:
I know that it would be a string, yes. Does the client library accept that or do you need to cast it?